Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -333,4 +416,5 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
 WI.MultiSidebar.Event = {
     MultipleSidebarsVisibleChanged: "multi-sidebar-multiple-sidebars-visible-changed",
     SidebarAdded: "multi-sidebar-sidebar-added",
+    HeightDidChange: "multi-sidebar-height-did-change"
```

Comment:
Please rename `HeightDidChange` -> `HeightChanged` to maintain consistency with other event names.